keyvaluestore.js

```diff
@@ -8,8 +8,11 @@ app.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
 const port = process.env.MICRO_KEYVALUESTORE_PORT || process.env.PORT || 9000;
 
 var keyValueStore = {}
+keyValueStore['masterendpoint'] = process.env.MICRO_MASTER_ENDPOINT || undefined;
```

Comment:
You don't need `|| undefined` part because `process.env.MICRO_MASTER_ENDPOINT` is undefined if not an env variable.